src/extension.ts

```diff
@@ -10,30 +10,31 @@ import * as vscode from 'vscode';
 import * as cpt from 'vscode-cpptools';
 import * as nls from 'vscode-nls';
 
-import {CMakeCache} from '@cmt/cache';
+import { CMakeCache } from '@cmt/cache';
```

Comment:
If you could avoid reformatting code files while making changes, it helps make the review easier for us.  Formatting changes are generally supposed to be done as a separate PR with no code changes.

Reply:
Got it, will revert. Just used recommended xaver.clang-format, as I format automatically when possible.